src/blog/2025/09/node-red-vs-n8n.md

```diff
@@ -0,0 +1,133 @@
+---
```

Comment:
If we build a competitive page, the URL will be `flowfuse.com/vs/<other-solution>`

cc @Yndira-E @PabloFilomeno83